### PR TITLE
* Fixes

### PR DESCRIPTION
**Pull request by Wagnerp**
- Fixes build errors

### CODE REVIEW

File: Source/Krypton Components/Krypton.Toolkit/Controls Toolkit/KryptonColorDialog.cs [hunk 1]
```diff
@@ -34,16 +34,6 @@ public string Title
                 ClickCallback = ClickCallback
             };
 
-        /// <summary>
```
Comment (Smurf-IV) [inline Source/Krypton Components/Krypton.Toolkit/Controls Toolkit/KryptonColorDialog.cs:37]: Why is this removed

File: Source/Krypton Components/Krypton.Toolkit/Controls Toolkit/KryptonFontDialog.cs [hunk 2]
```diff
@@ -31,16 +31,6 @@ public string Title
         /// that are currently installed on the system.
         /// </summary>
         public KryptonFontDialog() => _commonDialogHandler = new CommonDialogHandler(true);
-
```
Comment (Smurf-IV) [inline Source/Krypton Components/Krypton.Toolkit/Controls Toolkit/KryptonFontDialog.cs:34]: Why is this removed

File: Source/Krypton Components/Krypton.Toolkit/Controls Toolkit/KryptonPrintDialog.cs [hunk 3]
```diff
@@ -62,15 +62,6 @@ public string Title
                 ClickCallback = ClickCallback
             };
 
-        /// <summary>
```
Comment (Smurf-IV) [inline Source/Krypton Components/Krypton.Toolkit/Controls Toolkit/KryptonPrintDialog.cs:65]: Why is this removed

File: Source/Krypton Components/Krypton.Toolkit/Controls Toolkit/KryptonColorDialog.cs (hunk 1 above)
Comment (Wagnerp) [reply Source/Krypton Components/Krypton.Toolkit/Controls Toolkit/KryptonColorDialog.cs:37]: It had a duplicate entry, so it wouldn't build

File: Source/Krypton Components/Krypton.Toolkit/Controls Toolkit/KryptonFontDialog.cs (hunk 2 above)
Comment (Wagnerp) [reply Source/Krypton Components/Krypton.Toolkit/Controls Toolkit/KryptonFontDialog.cs:34]: It had a duplicate entry, so it wouldn't build

File: Source/Krypton Components/Krypton.Toolkit/Controls Toolkit/KryptonPrintDialog.cs (hunk 3 above)
Comment (Wagnerp) [reply Source/Krypton Components/Krypton.Toolkit/Controls Toolkit/KryptonPrintDialog.cs:65]: It had a duplicate entry, so it wouldn't build
